pages/common/b2-tools.md

```diff
@@ -0,0 +1,36 @@
+# b2-tools
+
+> Access all features of Backblaze B2 Cloud Storage easily.
+> More information: <https://www.backblaze.com/docs/cloud-storage-command-line-tools>.
+
+- Access your account:
+
+`b2 authorize_account {{key_id}}`
+
+- List the existing buckets in your account:
+
+`b2 list_buckets`
+
+- Create a bucket, provide the bucket name, and access type (e.g. allPublic or allPrivate):
+
+`b2 create_bucket {{bucket_name}} {{allPublic|allPrivate}}`
```

Comment:
```suggestion
`b2 create_bucket {{bucket_name}} {{allPublic|allPrivate|...}}`
```
Are there other options that allPublic and allPrivate?

Reply:
Only two options available.